src/components/pages/Home.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Home = () => (
+  <>
+    <h2>Welcome to our page</h2>
+    <p>
+      Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+      Quas magni ab expedita natus dolor architecto, ducimus
+      reiciendis voluptate consequatur cumque, maiores iusto temporibus,
+      tempora reprehenderit autem. Officia culpa eligendi cumque.
+    </p>
+    <p>
+      Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+      Quas magni ab expedita natus dolor architecto, ducimus
+      reiciendis voluptate consequatur cumque, maiores iusto temporibus,
+      tempora reprehenderit autem. Officia culpa eligendi cumque.
+    </p>
```

Comment:
- [ ] [OPTIONAL] Please try as much as possible to personalize the home page. This will make your project more professional ☺️.